javascript/builtins/AggregateError.json

```
@@ -142,7 +142,8 @@
             "mdn_url": "https://developer.mozilla.org/docs/Glossary/Serializable_object",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "98",
+                "notes": "`AggregateError` serializes to the `Error` type, without additional properties."
```

Given that `AggregateError` is already given as context through the parent feature and the feature description, can we make this more concise?

```suggestion
                "notes": "`Serializes to the `Error` type, without additional properties."
```

---

I think this clipped style, where the note is not a complete sentence looks pretty bad on caniuse. On MDN, it's OK; the text is never that far from the name of the feature. But on caniuse, the footnote is rather distant and it looks kinda lazy and incomplete.